crates/oci-tar-builder/Cargo.toml

```diff
@@ -18,6 +18,8 @@ serde = { workspace = true }
 serde_json = { workspace = true }
 clap = { version = "4.5.7", features = ["derive"] }
 indexmap = "2.2.6"
+oci-wasm = { path = "../../../rust-oci-wasm/" }
```

Comment:
CI is failing because of this, I need to get a fix out to this project and will then update

Reply:
https://github.com/bytecodealliance/rust-oci-wasm/pull/7